civihr_default_theme/scss/partials/_menu.scss

```diff
@@ -69,8 +69,11 @@
 }
 
 .chr_header__home-menu {
+  color: $light;
```

Comment:
Are you also adjusting the menu color? Was it part of the ticket requirements? It wasn't mentioned in the PR